float64 mainPos
float64 jibPos
float64 rudderPos